python/hail/matrixtable.py

```diff
@@ -1029,6 +1029,10 @@ def select_entries(self, *exprs, **named_exprs):
         for e in exprs:
             all_exprs.append(e)
             analyze('MatrixTable.select_entries', e, self._entry_indices)
+            if not e._indices == self._entry_indices:
```

Comment:
Little confused by this but I'll keep looking into it.  Shouldn't this broadcast?

Reply:
this was a bug. You were previously able to get into scala with `ds.select_entries(ds.locus)` because a row field satisfies an entry field. We need to add more to `analyze` long term to ask whether broadcasting is OK. this is a fix for now

Reply:
Ah, I see, this is only on the (unnamed) exprs?  Got it.

Reply:
yep!